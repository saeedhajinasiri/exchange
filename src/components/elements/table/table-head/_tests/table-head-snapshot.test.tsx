import React from "react";
import {cleanup, render} from "test";
import TableHead from '../table-head'
import TableRow from '../../table-row'
import Table from "../../table";

afterEach(cleanup)

describe('Test TableHead UI', () => {
  it('Default', () => {
    const {asFragment} = render(<Table><TableHead/></Table>);
    expect(asFragment()).toMatchSnapshot();
  });

  it('Test with children prop', () => {
    const children = <TableRow/>
    const {asFragment} = render(<Table><TableHead>{children}</TableHead></Table>);
    expect(asFragment()).toMatchSnapshot();
  });
});
