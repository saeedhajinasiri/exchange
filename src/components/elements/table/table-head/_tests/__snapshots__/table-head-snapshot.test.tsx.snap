// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test TableHead UI Default 1`] = `
<DocumentFragment>
  <table
    class="w-full"
  >
    <thead
      class="table-header-group"
    />
  </table>
</DocumentFragment>
`;

exports[`Test TableHead UI Test with children prop 1`] = `
<DocumentFragment>
  <table
    class="w-full"
  >
    <thead
      class="table-header-group"
    >
      <tr
        class="border-b border-grey-100 border-opacity-20 hover:bg-opacity-75 bg-white border-grey-light"
      />
    </thead>
  </table>
</DocumentFragment>
`;
